### PR TITLE
hiding duo mode for item create view

### PR DESCRIPTION
**Pull request by d88naimi**
Before
Screen Shot 2022-01-11 at 3 33 21 PM

After
Screen Shot 2022-01-11 at 3 33 08 PM

### CODE REVIEW

File: src/apps/content-editor/src/app/views/ItemCreate/ItemCreate.js [hunk 1]
```diff
@@ -63,6 +65,10 @@ export default function ItemCreate() {
     }
   }, [modelZUID, item]);
 
+  useEffect(() => {
+    dispatch(actions.setDuoMode(false));
```
Comment (shrunyan) [inline src/apps/content-editor/src/app/views/ItemCreate/ItemCreate.js:69]: won't this turn it off though? meaning anytime you create a new item you will be booted out of duomode and forced to toggle back on?

Comment (d88naimi) [reply src/apps/content-editor/src/app/views/ItemCreate/ItemCreate.js:69]: @stuart This was the best solution I had in mind beside trying to display none a lot of elements while in Item create view, unless you have a better route?

Comment (shrunyan) [reply src/apps/content-editor/src/app/views/ItemCreate/ItemCreate.js:69]: You will need to lift the DuoMode preview out of `Editor.js` and into https://github.com/zesty-io/manager-ui/blob/f6c5d14fd234848b9bf17ce8ebd13790bb18aad2/src/apps/content-editor/src/app/views/ItemEdit/Content/Content.js

Comment (d88naimi) [reply src/apps/content-editor/src/app/views/ItemCreate/ItemCreate.js]: @shrunyan  I have reevaluated the styles and got duo mode to hide correctly for item create with out touching state.

**Comment by d88naimi** [comment]
@shrunyan ready for review thank you.

Screen Shot 2022-01-12 at 1 19 13 PM

**Comment by d88naimi** [comment]
Verified with these new edits it does not cost extra on the network when using display: none

**Pull request merged**